demo.go

```diff
@@ -0,0 +1,92 @@
+package main
+
+import (
+	"fmt"
+)
+
+func main() {
+	mlp := NewMLP(2, []int{8, 8, 4, 1})
+
+	x_train, y_train, err := loadData("data.csv", 2)
+	if err != nil {
+		fmt.Print(err.Error())
```

Comment:
Lets just do a panic here.
```suggestion
		panic(err)
```